### PR TITLE
feat: added enviroment variables

### PR DESCRIPTION
**Pull request by L1ght-time**

### CODE REVIEW

File: src/pages/about/index.tsx [hunk 1]
```diff
@@ -1,11 +1,25 @@
 import type { NextPage } from 'next';
 import { FormattedMessage } from 'react-intl';
+import process from 'process';
 
-const About: NextPage = () => {
+export async function getStaticProps() {
+  const response = await fetch(`${process.env.API_HOST}/about`);
```
Comment (mykyta-lazurenko) [inline src/pages/about/index.tsx:6]: If you want use fetch by default, use ky for creating instance and reuse for all requests.

File: src/pages/api/db/about.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const ABOUT = {
+  title: 'ABOUT',
```
Comment (mykyta-lazurenko) [inline src/pages/api/db/about.ts:2]: It's `localeId`? What about internationalisation?

Comment (L1ght-time) [reply src/pages/api/db/about.ts:2]: I know, it's a temporary decision

Comment (L1ght-time) [reply src/pages/api/db/about.ts:2]: I know, it's temporary decision

File: src/pages/about/index.tsx (hunk 1 above)
Comment (L1ght-time) [reply src/pages/about/index.tsx]: I will implement it in the next task

**Pull request merged**